frontend/src/components/molecules/HighlightWithTwoButtons/style.css

```
@@ -0,0 +1,14 @@
+.datapass-container-two-btn {
```

this misses a rules :
```suggestion
.fr-alert + .datapass-container-two-btn {
  margin-top: 1rem;
}

.datapass-container-two-btn {
```